tests/models/test_gp_classification.py

```diff
@@ -798,10 +798,21 @@ def test_extra_ask_warns(self):
     def test_hyperparam_consistency(self):
         # verify that creating the model `from_config` or with `__init__` has the same hyperparams
 
-        m1 = GPClassificationModel(lb=[1, 2], ub=[3, 4])
+        m1 = GPClassificationModel(lb=[1, 2], ub=[3, 4], inducing_size=2)
```

Comment:
Why was this needed, does default the default not work? Defaults should be able to take over if the argument is not set.